dart/pubspec.lock

```diff
@@ -0,0 +1,311 @@
+# Generated by pub
```

Comment:
For published libraries Dart usually recommends to not publish the `pubspec.lock` (and it is usually put in `.gitignore` to keep it out of the repository). IIRC this won't be an issue if you publish it, but you may get a warning about it.